### PR TITLE
Updated Pandas lecture

### PR DESCRIPTION
**Pull request by trgardos**
Addresses #45

### CODE REVIEW

File: ds701_book/02B-Pandas.qmd [hunk 1]
```diff
@@ -1,77 +1,157 @@
...
-In general, we'll typically describe the rows in the dataframe as **items** and the columns as **features**.
+In general, we'll typically describe the rows in the dataframe as **items** 
+(or **observations** or **data samples**) and the columns as **features**.
 
 ```{python}
 df.columns
 ```
 
-Pandas allows you to use standard python __indexing__ using square brackets to refer to columns (e.g., features) in your dataframe:
+Pandas allows you to reference column similar to a python dictionary key,
```
Comment (sladenheim) [inline ds701_book/02B-Pandas.qmd:138]: small typo
> to reference a column

**Review by sladenheim — APPROVED** [review]
Nice! I like the change to NVDA and other tech companies as opposed to Yelp.

File: ds701_book/02B-Pandas.qmd (hunk 1 above)
Comment (trgardos) [reply ds701_book/02B-Pandas.qmd:138]: fixed

**Comment by trgardos** [comment]
Yeah, thought i'd make it a little more modern. Made me regret my investment choices however. :-)

**Pull request merged**